### ISSUE TITLE
A11y_Visual Studio Code_Docs Setup_Screen Reader: NVDA is reading two roles(button, link) for 'Edit' control

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/vscode-docs#4523]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11y_Visual Studio Code#A11ySev2#A11yTCS#A11yMAS#Chrome#DesktopWeb#Win10#Visual Studio Code Services#Screen Reader#WCAG1.3.1#NVDA#VisualStudioCode-Web-Apr2021

### Environment details:
Application Name: Visual Studio Code
URL:[Visual Studio Code](https://code.visualstudio.com/docs/setup/linux)
Windows Version: Version 20H2 (OS Build 19042.928)
Browser: Chrome Version 90.0.4430.85 (Official Build) (64-bit)
Screen Readers: NVDA

### Note:

- This issue is observed for 'Edit' control in all pages through out the application
- This issue is not observed with Narrator in Edge browser

### Repro Steps:

1. Turn on NVDA
2. Hit [Visual Studio Code](https://code.visualstudio.com/docs/setup/linux) link
3. Tab till Docs link in top navigation section and press enter
4. Tab till Set up in left navigation pane and select Linux/Mac OS
5. Tab till Edit control and verify that NVDA announces two roles(button, link) for 'Edit' control

### Actual Result:
NVDA is reading two roles(button, link) for 'Edit' control

### Expected Result:
NVDA should not read two roles(button, link) for 'Edit' control. Only one role should be defined to any control and so screen reader should also read only one role for any control

### User Impact:
Screen Reader users might get confused if multiple roles are defined for a control

### Attachment:

https://user-images.githubusercontent.com/83570638/117607277-41ead400-b179-11eb-936f-fbc408afe1c1.mp4

https://user-images.githubusercontent.com/83570638/117123807-1e104280-adb5-11eb-877e-28089bbc5376.mp4

**Comment by Ojeskey** [comment]
H

**Comment by kalpanah1** [comment]
#Closed;
Verified the issue in the below environment. The issue is fixed. Please find the attachment for reference.
[Uploading Bug_4523_Fixed.zip…]()
